examples/https.rs

```diff
@@ -0,0 +1,27 @@
+#[macro_use] extern crate nickel;
+extern crate hyper;
+
+#[cfg(feature = "ssl")]
+fn main() {
+    ssl::main();
```

Comment:
Hmmm ... you just made me discover I could use `use` inside functions ... So no, no particular reason.